Proteomics/RetentionTimePrediction/SSRCalc3.cs

```diff
@@ -0,0 +1,1486 @@
+            if (where > sqlen - 5) { return 0.65; }
+
+            return 1.0;
+        }
+
+        // ============================================================
+        // helicity1 adjust for short helices or sections - v 3 algorithm
+        //
+        private double helicity1(string sq)
+        {
+            if (NOHELIX1 == 1)
+                return 0.0;
+
+            string hc = sq; //helicity coded sq
+
+            /* Translator1 note:  notice lowercase 'z'.  This never appears in any patterns to which this
+               string is compared, and will never match any helicity patterns.
+            */
+            hc = hc.ReplaceAAs("PHRK", "z"); // Not L10N
+            hc = hc.ReplaceAAs("WFIL", "X"); // Not L10N
+            hc = hc.ReplaceAAs("YMVA", "Z"); // Not L10N
+            hc = hc.ReplaceAAs("DE", "O"); // Not L10N
+            hc = hc.ReplaceAAs("GSPCNKQHRT", "U"); // Not L10N
+
+            double sum = 0.0;
+            int sqlen = hc.Length;
+
+            // Translator1 note: this loop should be reviewed carefully
+
+            for (int i = 0; i < sqlen - 3; i++)
+            {
+                string hc4 = string.Empty, hc5 = string.Empty, hc6 = string.Empty;
+                double sc4 = 0.0, sc5 = 0.0, sc6 = 0.0;
+
+                if (hc.Substring(i).Length >= 6)
+                {
+                    hc6 = hc.Substring(i, 6);
+                    sc6 = 0.0;
+                    if (HlxScore6.ContainsKey(hc6))
+                    {
+                        sc6 = HlxScore6[hc6];
+                    }
+                }
+                if (sc6 > 0)
+                {
+                    double trmAdj6 = heli1TermAdj(hc6, i, sqlen);
+                    sum += (sc6 * trmAdj6);
+                    i = i + 1; //??
+                    continue;
+                }
+
+                if (hc.Substring(i).Length >= 5)
+                {
+                    hc5 = hc.Substring(i, 5);
+                    sc5 = 0.0;
+                    if (HlxScore5.ContainsKey(hc5))
+                    {
+                        sc5 = HlxScore5[hc5];
+                    }
+                }
+                if (sc5 > 0)
+                {
+                    double trmAdj5 = heli1TermAdj(hc5, i, sqlen);
+                    sum += (sc5 * trmAdj5);
+                    i = i + 1; //??
+                    continue;
+                }
+
+                if (hc.Substring(i).Length >= 4)
+                {
+                    hc4 = hc.Substring(i, 4);
+                    sc4 = 0.0;
+                    if (HlxScore4.ContainsKey(hc4))
+                    {
+                        sc4 = HlxScore4[hc4];
+                    }
+                }
+                if (sc4 > 0)
+                {
+                    double trmAdj4 = heli1TermAdj(hc4, i, sqlen);
+                    sum += (sc4 * trmAdj4);
+                    i = i + 1; //??
+                               // continue;
```

Comment:
remove commented `continue;` if not used

Reply:
done